### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+instead.
+
+Please consider helping us by reviewing other peoples pull requests as well:
+
+- pick up any PR at <https://coala.io/review>
+- review it (check <https://coala.io/reviewing> for more info)
+- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
+  of the review queue.
+
+The more you review, the more your score will grow at coala.io and we will
```
Comment (jayvdb) [inline PULL_REQUEST_TEMPLATE.md]: "at coala.io" - should be a link, via something like coala.io/score ?

Comment (sims1253) [inline PULL_REQUEST_TEMPLATE.md]: I don't think this needs a link. If someone is interested its 2 clicks away from them

**Comment by jayvdb** [comment]
ack 03e82da

**Comment by sims1253** [comment]
@rultor merge

**Pull request merged**